modules/addListModal.ts

```diff
@@ -0,0 +1,94 @@
+import { addListForm } from './domElements';
+//TODO: We should create a 'modal' directory, where we can share common logic
```

Comment:
We're repeating ourselves a lot in each of these modals, and it's a pattern we're using for the project at large (though that could change if we adapt routing to be more SPA-like) - as the note says, I should take a few minutes to plan a path forward for making a modal directory to more concisely handle and share modal logic and behavior.